tests/datasets/configuration/test_entity_loader.py

```diff
@@ -95,6 +99,22 @@ def test_entity_loader_for_enitity_with_column_mappers() -> None:
     assert len(function_call.parameters) == 1
     assert any(isinstance(param, Column) for param in function_call.parameters)
 
+    # Check that ColumnToIpAdress mapper was successfully loaded from config
```

Comment:
```suggestion
    # Check that ColumnToUserNullIf mapper was successfully loaded from config
```

Or if modified to be generic:
```suggestion
    # Check that ColumnToNullIf mapper was successfully loaded from config
```